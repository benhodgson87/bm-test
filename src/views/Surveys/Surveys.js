import React from 'react'

import MainHeader from '../../components/MainHeader'
import JobsHeader from '../../components/JobsHeader'
import Jobs from '../../components/Jobs'

const Surveys = () => (
  <div>
    <MainHeader />
    <JobsHeader />
    <Jobs />
  </div>
)

export default Surveys
