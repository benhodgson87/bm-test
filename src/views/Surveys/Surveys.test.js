import React from 'react'
import { shallow } from 'enzyme'

import Surveys from './Surveys'

describe('Surveys view component', () => {
  test('renders without crashing', () => {
    const wrapper = shallow(<Surveys />)
    expect(wrapper).toMatchSnapshot()
  })
})
