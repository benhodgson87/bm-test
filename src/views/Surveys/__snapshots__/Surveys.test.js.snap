// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Surveys view component renders without crashing 1`] = `
<div>
  <MainHeader />
  <JobsHeader />
  <Connect(JobsContainer) />
</div>
`;
